src/pages/HomeParticipant.vue

```diff
@@ -9,19 +9,42 @@
         <h2 v-if="adminInfo?.length == 1" class="p-float-label dropdown-container">
           {{ adminInfo.at(0).name }}
         </h2>
-        <div v-if="adminInfo?.length > 1" class="p-float-label dropdown-container">
+        <div
+          v-if="adminInfo?.length > 1"
+          class="flex flex-row justify-center align-items-center p-float-label dropdown-container gap-4"
+        >
           <PvDropdown
             v-model="selectedAdmin"
             :options="adminInfo ?? []"
             option-label="name"
             input-id="dd-assignment"
             data-cy="dropdown-select-administration"
           />
+          <PvInputSwitch v-model="showOptionalAssessments" data-cy="switch-show-optional-assessments" />
+          <p>Show Optional Assessments</p>
           <label for="dd-assignment">{{ $t('homeParticipant.selectAssignment') }}</label>
         </div>
         <div class="tabs-container">
           <ParticipantSidebar :total-games="totalGames" :completed-games="completeGames" :student-info="studentInfo" />
-          <GameTabs :games="assessments" :sequential="isSequential" :user-data="userData" />
+          <Transition name="fade" mode="out-in">
+            <p v-if="showOptionalAssessments && optionalAssessments.length === 0" class="m-auto">
+              No optional games have been assigned.
+            </p>
```

Comment:
Could we just conditionally show the optional games switch? So that the switch itself doesn't show up if there are no optional games.